### DISCUSSION TITLE
pivCCU?

### DISCUSSION THREAD

**Discussion by theghostronaut** [thkl/hap-homematic#336] [Q&A] [answered]
Ist das Plugin kompatibel mit pivCCU auf einem Raspi?

**Comment by AtlanArk** [comment, marked as answer]
Ja, bei mir geht es ohne Probleme. Direkt nach der Anleitung 
Mfg

**Reply by ChrWi** [reply]
Klar, läuft super. Bei Fragen einfach melden.
Bei mir: Pi 4 mit Homebridge und HomeMatic 3.59.6 (piVCCU3, HM-MOD-RPI-PCB)
Homematic-HAP, HmIP-PSM, HmIP-PS, HmIP-BSM, HmIP-BROLL x8
Garagentor (HMIP-SWDO, HM-LC-Sw1-Pl-CT-R1)
